docs/coin/get-currency.md

```diff
@@ -0,0 +1,63 @@
+---
+sidebar_position: 3
+---
+
+# Get Currency
+
+## 1. version control
+
+| Version  | Date        | Description of Changes |
+| -------- | ----------- | ---------------------- |
+| CES V1.0 | May 3, 2023 | Initial Release        |
+
+## 2. Overview
+
+#### In cryptocurrency exchange system coin/currency can get details by ID or Symbol or Name.
+
+
+## 3. API
+
+### Get Coin/Cuerrncy Request example:
+
+```json
+{
+   "ID":"123e4567-e89b-12d3-a456-426614174000",
+   "Symbol":"BTC",
+   "Name":"Bitcoin"
+}
+```
+
+### Get Coin/Cuerrncy Response example:
```

Comment:
Extend a section including Enum fields and string values (with uppercase)

Reply:
updated